contracts/PushComm/PushCommV3.sol

```diff
@@ -689,21 +689,33 @@ contract PushCommV3 is Initializable, PushCommStorageV2, IPushCommV3, PausableUp
      */
     // Should be only admin
     // Should only bridge NTT TOKENS FROM COMM TO CORE on ethereum
-    function transferFeePoolToCore(uint256 _amount) external payable onlyPushChannelAdmin {
+    function transferFeePoolToCore(CrossChainRequestTypes.CrossChainFunction functionType, uint256 _amount, uint256 gasLimit) external payable onlyPushChannelAdmin {
         if (PROTOCOL_POOL_FEE < _amount) {
             revert Errors.InsufficientFunds();
         }
-
-        bytes32 recipient = bytes32(uint256(uint160(EPNSCoreAddress)));
-
+        uint256 messageBridgeCost = quoteMsgRelayCost(WORMHOLE_RECIPIENT_CHAIN, gasLimit);
         uint256 tokenBridgeCost = quoteTokenBridgingCost();
-        if (msg.value < tokenBridgeCost) {
+
+        if (msg.value < (messageBridgeCost + tokenBridgeCost)) {
             revert Errors.InsufficientFunds();
         }
 
         PROTOCOL_POOL_FEE = PROTOCOL_POOL_FEE - _amount;
```

Comment:
GAS#1: PROTOCOL_POOL_FEE can also be stored in memory and then updated in the last